go.mod

```diff
@@ -308,7 +308,8 @@ replace github.com/prometheus/client_golang => github.com/prometheus/client_gola
 replace golang.org/x/net => golang.org/x/net v0.5.0
 
 // https://issues.redhat.com/browse/OCPBUGS-8119
-replace github.com/containerd/containerd => github.com/containerd/containerd v1.5.18
+// https://issues.redhat.com/browse/OCPBUGS-27507
```

Comment:
Nit: There are 2 bug fixes in the comments. Is that intended?

Reply:
Yes, but I can drop the oldest if that's preferable.